src/components/Callout/CalloutContent.tsx

```diff
@@ -65,16 +64,32 @@ export class CalloutContent extends BaseComponent<ICalloutProps, ICalloutState>
   }
 
   public render() {
-    let { className, targetElement, isBeakVisible, beakStyle, children } = this.props;
+    let { className, targetElement, isBeakVisible, beakStyle, children, beakWidth } = this.props;
     let { positions, slideDirectionalClassName } = this.state;
+    let beakStyleWidth = beakWidth;
+
+    if (beakStyle === 'ms-Callout-smallbeak') {
```

Comment:
Add a comment here that this is to support deprecated compatibility and should be removed @ 1.0.0.